examples/angular/src/environments/environment.development.ts

```diff
@@ -1,5 +1,7 @@
-export const environment = {
-    dotcmsUrl: "http://localhost:8080",
-    authToken: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhcGlkZTU4MDZmNS1iZDZlLTRkMTYtOWVhZi1jM2ZmMTRiYzIwZmUiLCJ4bW9kIjoxNzE3NjE4MDAwMDAwLCJuYmYiOjE3MTc2MTgwMDAsImlzcyI6IjhmOGJjZWUxZTUiLCJsYWJlbCI6ImtkIiwiZXhwIjoxODEyMTcxNjAwLCJpYXQiOjE3MTc2MTgwMDAsImp0aSI6IjNiNDI1NjQ5LTNlNDUtNGM2My1iOTE3LWE5MGUwMGM4M2UxYiJ9.niN9ZWPGxit5JuL94t89gNPpVSxWV--9OkJpwGjInZQ",
-    siteId: "true",
+const environment = {
+  dotcmsUrl: 'http://localhost:8080',
+  // This must be a read-only token.
```

Comment:
"This auth token will be exposed in the client-side requests, for security purposes, make sure is read-only. More info: https://www.dotcms.com/docs/latest/permissions"